android-project/app/src/main/java/org/libsdl/app/SDLActivity.java

```diff
@@ -2110,11 +2119,9 @@ public void run() {
             Log.v("SDL", "modify thread properties failed " + e.toString());
         }
 
-        Log.v("SDL", "Running main function " + function + " from library " + library);
-
-        SDLActivity.nativeRunMain(library, function, arguments);
```

Comment:
The library, function, and arguments variables are no longer used, right?
Does it make sense to set the thread priority here, or in nativeInitMainThread() or main()?

Reply:
> The library, function, and arguments variables are no longer used, right?

Good catch.

> Does it make sense to set the thread priority here, or in nativeInitMainThread() or main()?

`android.os.Process.setThreadPriority` is a Java API, so it makes sense to call it from java code.